Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Shotguns/marine_shotguns.yml

```diff
@@ -116,5 +116,93 @@
       rmc-aslot-stock: -0.375, -0.029
       rmc-aslot-underbarrel: 0.25, -0.31
 
+- type: entity
+  parent: [ CMBaseWeaponRifle, RMCBaseAttachableHolder ]
```

Comment:
Why does this inherit from the base rifle when it's in the shotgun file? I understand that CM has it under rifles for some bloody reason, but we have separate files for this for a reason. This gun is functionally a shotgun. It should go under shotguns. We can just adjust any stats that changes.

Reply:
you can split the prototype in to two different prototypes (one with the components you dont need, and the other one